apps/connections/index.jsx

```diff
@@ -91,4 +93,9 @@ var Connections = React.createClass({
   }
 })
 
-module.exports = Connections
+ipfsapp.define({
+  init: function (ipfs) {
+    React.render(<Connections ipfs={ipfs} />,
+                 document.getElementById('mount'))
+  }
+})
```

Comment:
if we could match the typical 

```js
module.exports = function() {
}
// or 
module.exports = {} 
```
type of thing, that may be nice. maybe not easy? this may be ok. impressions @diasdavid?